spec/cloud-functions.spec.ts

```diff
@@ -204,19 +178,6 @@ describe('makeParams', () => {
   };
   const cf = makeCloudFunction(args);
 
-  it('should construct params from the event resource of legacy events', () => {
-    const testEvent: LegacyEvent = {
-      resource: 'projects/_/instances/pid/ref/a/nested/b',
-      eventType: 'legacyEvent',
-      data: 'data',
-    };
-
-    return expect(cf(testEvent)).to.eventually.deep.equal({
-      foo: 'a',
-      bar: 'b',
-    });
-  });
-
   it('should construct params from the event resource of new format events', () => {
```

Comment:
should this just be updated to remove `new format` since there's no legacy anymore?